libraries/abstractions/secure_sockets/cellular/iot_secure_sockets.c

```diff
@@ -175,6 +183,17 @@ static int32_t prvCheckSetSockOptParams( Socket_t xSocket,
                                          const void * pvOptionValue,
                                          TickType_t * pSockTimeout );
 
+#if ( CELLULAR_SUPPORT_GETHOSTBYNAME == 0 )
+    static const char * prvReverseLookup( uint32_t ipAddress );
+    static uint32_t prvLookup( const char * pcHostName );
+#endif /* if ( CELLULAR_SUPPORT_GETHOSTBYNAME == 0 ) */
+
+/*-----------------------------------------------------------*/
+
+#if ( CELLULAR_SUPPORT_GETHOSTBYNAME == 0 )
+    static char _dnsCache[ GETHOSTBYNAME_CACHE_SIZE ][ CELLULAR_IP_ADDRESS_MAX_SIZE + 1 ];
```

Comment:
We no longer start internal variable or function names with underscore as those are reserved by the language. We can address this in one go in all the Cellular code.

Reply:
A new PR will be created for all the internal variable or function names with underscore.